frontend/src/components/RecordsList.tsx

```diff
@@ -29,6 +29,7 @@ import { ErrorMessage, LoadingSpinner, Toolbar } from "../shared/tableElements";
 interface IRecordsListProps {
   colDefs: ColDef[];
   dataName: DataName;
+  enableInfiniteScroll?: boolean;
```

Comment:
As a refresher, `RecordsList` is the component that renders Requests, Patients, and Cohorts pages. By default, this will be on. For the Cohorts page, we will turn this off to enable client-side filtering.

This will make the Cohorts page fetch the entire dataset to the browser on each render, but I'm not concerned about performance because the Cohorts dataset is tiny.